apps/console/src/features/roles/components/edit-role/edit-role-basic.tsx

```diff
@@ -245,6 +260,16 @@ export const BasicRoleDetails: FunctionComponent<BasicRoleProps> = (props: Basic
                     </ConfirmationModal>
                 )
             }
+            {
+                showDeleteErrorConnectedAppsModal && (
+                    <RoleDeleteErrorConfirmation
+                        selectedRole={ role }
+                        isOpen={ showDeleteErrorConnectedAppsModal }
```

Comment:
If there's already an `isOpen` prop that controls the modal visibility, why do we need another guard clause in L264?